articles/active-directory-b2c/date-transformations.md

```diff
@@ -167,7 +167,7 @@ Get the current UTC date and time and add the value to a ClaimType.
 
 ## DateTimeComparison
 
-Determine whether one dateTime is later, earlier, or equal to another. The result is a new boolean ClaimType boolean with a value of `true` or `false`.
+Determine whether one dateTime is later, earlier, or equal to another. The result is a new boolean ClaimType with a value of `true` or `false`.
```

Comment:
Please change the ClaimType to claim type

Reply:
Thanks @yoelhor - it is referenced as `ClaimType` in all other sections of this doc. Do you want me to update all the section summaries from ClaimType to claim type? 
Just want to keep things consistent 😃

Reply:
Any resolution on this point?